plugin/tls/tls.go

```diff
@@ -25,6 +26,24 @@ func setup(c *caddy.Controller) error {
 	return nil
 }
 
+func hardenTLS(tls *ctls.Config) {
+	os.Setenv("GODEBUG", "tls13=1")
```

Comment:
This looks odd and brittle, it also overwrites GODEBUG which is something we don't want. Please remove this line.

Reply:
Unfortunately, this seems the only way to activate TLS 1.3 in Go right now.

I don't know how to do it cleaner.

Reply:
I could change it to
`os.Setenv("GODEBUG", os.Getenv("GODEBUG")+",tls13=1")`

At least I won't overwrite the variable.

I took the info from here https://github.com/golang/go/issues/30055

Reply:
Should we wait for the default TLS 1.3 support in Go fow now? 
I am not sure why they choose the opt-in via GODEBUG